frontend/src/metabase/visualizations/components/ColumnSettings.jsx

```diff
@@ -67,22 +66,11 @@ export function hasColumnSettingsWidgets({ value, ...props }) {
   return getWidgets({ storedSettings, ...props }).length > 0;
 }
 
-const ColumnSettings = ({
-  value,
-  variant = "default",
-  forcefullyShowHiddenSettings = false,
-  ...props
-}) => {
+const ColumnSettings = ({ value, variant = "default", ...props }) => {
   const storedSettings = value || {};
   const widgets = getWidgets({ storedSettings, ...props });
   const extraWidgetProps = {};
```

Comment:
You can remove this line since this was only being used to pass a `hidden = false` value (that was being set by the `forcefullyShowHiddenSettings` check)